src/base/bittorrent/session.cpp

```diff
@@ -155,6 +155,19 @@ namespace
 
         return expanded;
     }
+
+    QStringList findAllFiles(const QString &folderPath)
+    {
+        QStringList files;
```

Comment:
One can ignore this situation, of course, just to get a file I/O error a bit later. Additionally, in this very case we need to determine whether we have an incomplete torrent. Thus for us a broken symbolic link is equal to an absent file.